config/rbac/submariner-globalnet/cluster_role.yaml

```diff
@@ -5,6 +5,13 @@ metadata:
   creationTimestamp: null
   name: submariner-globalnet
 rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    verbs:
+      - get
+      - list
```

Comment:
GN only needs to read `ConfigMaps` in the operator namespace so we should put this in _role.yaml_.